Tests/test_dynamic_loading.py

```diff
@@ -82,6 +82,11 @@ def test_can_import_trigger(self):
         triggers = self.get_package_in_folder(self.trigger_dir)
         package_name = "trigger"
         for trigger in triggers:
+
+            # snowboy is only compiled for python2
```

Comment:
[Here ](https://github.com/kalliope-project/kalliope/tree/master/kalliope/trigger/snowboy)you have two folder. We'll need to create subfolder in armv7 and x86_64 called python2 and python 3 and place your new compiled so in py3 and move the current so in python 2.
Of course, after that, the code need to be updated too in order to handle those 2 sub folder.
You can add a new attribute to the settings model like we did with "machine" [here](https://github.com/kalliope-project/kalliope/blob/master/kalliope/core/Models/Settings.py#L28). This is where we store the "armv7" or x86_64" info about the current machine which is executing Kalliope.